CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.12.3]
+### Changed
+- added `throttleKeypresses`, which in combination with `throttle` prevents to stop throttling by `keyup`
+- usefull in case of a fast pressing keys like arrows at a page with a lot of complex elements
```

Comment:
> usefull in case of a fast pressing keys like arrows at a page with a lot of complex elements

By "complex elements", do you mean elements that require more memory and CPU usage?

If you need to include this point, I'd phrase it _something_ like this:
- Useful when there are issues with the performance of handling rapidly firing navigational events

Also, "usefull" is misspelled here. Does your IDE have a spellchecker you could enable? 😄

Reply:
I'm using [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker). Since it's VS Code is not IDE, but only editor it's sometimes doesn't work properly.

Reply:
https://github.com/NoriginMedia/react-spatial-navigation/pull/67/commits/296cd18473ac31d0b22707be3188168646677c63